lib/feeder_importer.rb

```diff
@@ -0,0 +1,72 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+
+class FeederImporter
```

Comment:
see long note above. tl;dr made the standalone first, then optimized it to create the api importer.

I can imagine cases where I might want to be able to import from my local to some test/prod system in a synch way.